Makefile

```
@@ -49,10 +58,15 @@ PANDOC_EPUB_OPTIONS=--to epub3
 
 # Targets and dependencies
 
-.PHONY: all clean
+.PHONY: all clean combined.epub
```

The target name needs to change either here or in the definition of the target.

```suggestion
.PHONY: all clean epub
```